src/ankita/WhileLoopExamples.java

```diff
@@ -0,0 +1,48 @@
+package ankita;
+
+public class WhileLoopExamples {
+	void prinEvenNumber(int start,int end) {
+		int i=start;
+		System.out.println("Even Numbers between "+ start +"&"+end);
+		while(i<=end) {
```

Comment:
in while you can directly give start and will increase value of start, no need to create extra variable. think on this.

Reply:
Yes. Got it.